magento/magento2
# Fix translation retrieval

### Description (*)
Translationstrings are collected in the static content deploy step with configurable regex-patterns (for the js-tranlsations.json-file - all translations available within a js context (js, ko-templates)).

when bundling of js files is enabled, all html-ko templates do get bundled and attribute-quotes do get escaped. this breaks the current js-file retrieval pattern for translation args

### Manual testing scenarios (*)

1. add translation for <Your cart> in any language
2. js bundling activated, set locale for the store to where the translation is added
3. static content deploy
4. go to checkout 2. page in luma theme
5. check if Your cart (right summary box headline) is translated

### Contribution checklist (*)
